pixi.toml

```diff
@@ -116,7 +116,7 @@ cmd = "isort ."
 
 [dependencies]
 pandoc = ">=3.7.0.2,<4"
-python = ">=3.10"
+python = ">=3.10,<3.14"
```

Comment:
`pixi` resolves this before PyPI dependencies. So if there is no upper bound, `pixi` has no reason not to install Python 3.14. Then, when it's time to install the PyPI dependencies, there seems to be one that doesn't work with Python 3.14.